jina/peapods/remote.py

```diff
@@ -49,29 +53,52 @@ def loop_body(self):
         if self.remote_id:
             self.logger.success(f'created remote {self.api.kind} with id {colored(self.remote_id, "cyan")}')
             self.set_ready()
-            self.api.log(self.remote_id)
+            self.api.log(self.remote_id, self.is_shutdown)
         else:
             self.logger.error(f'fail to create {typename(self)} remotely')
-            self.is_shutdown.set()
 
     def delete_remote(self):
         if hasattr(self, 'api') and self.api.is_alive and self.remote_id:
             self.api.delete(self.remote_id)
```

Comment:
After this fix, this function is deprecated.

Reply:
Done! Thx